os/os.module.go

```diff
@@ -0,0 +1,224 @@
+	} else {
+		if len(args) == 1 {
+			if reflect.TypeOf(args[0]).String() == "py.String" {
+				key = args[0]
+			} else {
+				return nil, py.ExceptionNewf(py.TypeError, "Expected argument of type string")
+			}
+			default_ = py.None
+		} else if len(args) == 2 {
+			if reflect.TypeOf(args[0]).String() == "py.String" && reflect.TypeOf(args[1]).String() == "py.String" {
+				key = args[0]
+				default_ = args[1]
+			} else {
+				return nil, py.ExceptionNewf(py.TypeError, "Expected argument of type string")
+			}
+		}
+		var res py.Object // hold the result value
+		res, err = getEnvVariables().M__getitem__(key)
+		if err != nil {
+			return default_, nil
+		}
+		return res, nil
+	}
+}
+
+// get the current process' pid
+func getpid(self py.Object, args py.Tuple) (py.Object, error) {
+	return py.Int(os.Getpid()), nil
+}
+
+// Set the environment variable named key to the string value.
+func putenv(self py.Object, args py.Tuple) (py.Object, error) {
+	if len(args) == 2 {
+		if reflect.TypeOf(args[0]).String() == "py.String" && reflect.TypeOf(args[1]).String() == "py.String" {
+			_k, err := py.ReprAsString(args[0])
+			if err != nil {
+				return nil, py.ExceptionNewf(py.TypeError, "Unable to parse string") // this will never execute
+			}
+			key := strings.ReplaceAll(_k, "'", "") // required
+			_v, err := py.ReprAsString(args[1])
+			if err != nil {
+				return nil, py.ExceptionNewf(py.TypeError, "Unable to parse string") // this will never execute
+			}
+			value := strings.ReplaceAll(_v, "'", "")
+
+			err = os.Setenv(key, value)
+			if err != nil {
+				return nil, py.ExceptionNewf(py.OSError, "Unable to set enviroment variable")
+			}
+			return py.None, nil
+		} else {
+			return nil, py.ExceptionNewf(py.TypeError, "Expected 2 arguments of type string")
+		}
+	} else {
+		return nil, py.ExceptionNewf(py.TypeError, "Expected 2 arguments of type string")
+	}
+}
+
+// Unset (delete) the environment variable named key.
+func unsetenv(self py.Object, args py.Tuple) (py.Object, error) {
+	if len(args) == 1 {
+		if reflect.TypeOf(args[0]).String() == "py.String" {
+			_k, err := py.ReprAsString(args[0])
+			if err != nil {
+				return nil, py.ExceptionNewf(py.TypeError, "Unable to parse string") // this will never execute
+			}
+			key := strings.ReplaceAll(_k, "'", "") // required
+			err = os.Unsetenv(key)
+			if err != nil {
+				return nil, py.ExceptionNewf(py.OSError, "Unable to unset enviroment variable")
+			}
+			return py.None, nil
+		} else {
+			return nil, py.ExceptionNewf(py.TypeError, "Expected 1 argument of type string")
+		}
+	} else {
+		return nil, py.ExceptionNewf(py.TypeError, "Expected 1 argument of type string")
+	}
+}
+
+// os._exit() immediate program termination; unline sys.exit(), which raises a SystemExit, this function will termninate the program immediately.
+func _exit(self py.Object, args py.Tuple) (py.Object, error) { // can never return
+	if len(args) == 0 {
+		os.Exit(0)
+		return nil, nil
```

Comment:
since os.Exit doesn't return you can avoid all the `return nil, nil` and have just one at the end of this method.